modules/nextflow/src/main/groovy/nextflow/util/ConfigHelper.groovy

```diff
@@ -277,10 +278,6 @@ class ConfigHelper {
         result.toString()
     }
 
-    static String toCanonicalString(Map map, boolean sort=false) {
-        toCanonicalString(map.toConfigObject(), sort)
-    }
-
     static String toPropertiesString(ConfigObject config, boolean sort=false) {
```

Comment:
Think should stay

Reply:
The method is never used. However, I will undo the deletion but add a warning. Should I mark the method as deprecated?

Reply:
Why not used? I see used in several places https://github.com/nextflow-io/nextflow/pull/2496/files#diff-bbf841c8f39ba480213c8a171b78929cc4e68a38718b4386fa458f8012973705R327

Reply:
Here `config `is a `ConfigObject`, which is processed by this method: https://github.com/nextflow-io/nextflow/blob/67435473b9a5555edc208332bbb28fd8deb3a2ca/modules/nextflow/src/main/groovy/nextflow/util/ConfigHelper.groovy#L274-L278
The method with signature `toCanonicalString(Map map, boolean sort=false)` is never used.

Reply:
I see. In any case, it may be useful in the future. No need to deprecate it. 👍